libft/Makefile

```
@@ -27,7 +27,7 @@ SOURCES_MANDATORY = \
 			ft_memchr.c      ft_striteri.c       ft_toupper.c \
 			ft_memcmp.c      ft_strjoin.c \
 			ft_memcpy.c      ft_strlcat.c \
-			ft_free_split.c \
+			ft_free_split.c	 ft_strcat.c\
```

새로운 함수 추가는 새 줄에 부탁드려요
```suggestion
			ft_free_split.c \
			ft_strcat.c\
```

---

수정 완료했습니다